SVETLANA_SAT/script.js

```diff
@@ -0,0 +1,154 @@
+
+//  -TASK 1- Split and Merge
+
+function splitAndMerge(str, sp) {
+  return str.split(" ").join("").split("").join(sp);
+}
```

Comment:
@Lana2683 Please pay attention to description of the task and especially to the second example. We need to place separator only between letters in words, not between words

Reply:
fixed the task

Reply:
Thanks